client/pull.go

```diff
@@ -76,7 +76,7 @@ func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpt) (_ Ima
 	}
 
 	span.SetAttributes(
-		tracing.Attribute("image.ref", ref),
+		tracing.Attribute(tracing.AttrImageRef, ref),
```

Comment:
I don’t think we need to include this refactor in your change, since I noticed there are many attribute names that don’t have dedicated constant variables.